### PR TITLE
feat: Update extension banner experiment

### PR DESCRIPTION
**Pull request by AmarTrebinjac**
## Changes

Update extension experiment banner with new changes. Unsure if this needs a logEvent. Asked Ido for clarification.

Changes from quick sync with Tsahi:
* Avatar and text same line
* Adjust gap between sections
* Let left side grow
* Only for experiment

Added icon size 48 because this is the size from the design, but the icons in the banner were slightly bigger before.

examples:
Screenshot 2024-12-20 at 01 00 48
Screenshot 2024-12-20 at 01 01 36
Screenshot 2024-12-20 at 01 01 54
Screenshot 2024-12-20 at 01 02 18

## Events

Did you introduce any new tracking events?

## Experiment

Did you introduce any new experiments?

## Manual Testing

> [!CAUTION]
> Please make sure existing components are not breaking/affected by this PR

### Preview domain
https://mi-703.preview.app.daily.dev

MI-703 #done

### CODE REVIEW

File: packages/shared/src/components/auth/OnboardingHeadline.tsx [hunk 1]
```diff
@@ -10,6 +11,7 @@ interface ClassName {
 }
 
 interface OnboardingHeadlineProps {
+  avatar?: ReactElement;
```
Comment (sshanzel) [inline packages/shared/src/components/auth/OnboardingHeadline.tsx:14]:
```suggestion
  avatar?: ReactNode;
```

File: packages/shared/src/components/auth/AuthenticationBanner.tsx [hunk 2]
```diff
@@ -14,75 +14,108 @@ import {
...
+    feature: featurePostBannerExtensionPrompt,
+    shouldEvaluate: isCompatibleBrowser,
+  });
+
   const { showLogin } = useAuthContext();
 
   return (
     <BottomBannerContainer
       className={classNames(
-        'gap-24 border-t border-accent-cabbage-default py-10 shadow-3 laptopL:gap-32',
```
Comment (sshanzel) [inline packages/shared/src/components/auth/AuthenticationBanner.tsx:28]: Are these classes removed as intended?

**Review by sshanzel — APPROVED** [review]
Looks good to me! 🚀

**Review by rebelchris — APPROVED** [review]
Overall looks good, trust you tested it visually in both states :)

File: packages/shared/src/components/auth/AuthenticationBanner.tsx (hunk 2 above)
Comment (AmarTrebinjac) [reply packages/shared/src/components/auth/AuthenticationBanner.tsx:28]: @sshanzel Yeah, design has 40px gap between the sections and Tsahi specifically mentioned the gap sized in the sync 😄

**Pull request merged**